frank-doc-doclet/src/main/java/org/frankframework/frankdoc/model/FrankElement.java

```diff
@@ -211,6 +217,34 @@ private void assembleParsedJavaDocTags(FrankClass clazz, String tagName, Consume
 		}
 	}
 
+	private void handleLabels(FrankClass clazz, LabelValues labelValues) {
+		List<FrankAnnotation> annotationsForLabels = Arrays.asList(clazz.getAnnotations()).stream()
+				.filter(a -> a.getAnnotation(LABEL) != null)
+				.collect(Collectors.toList());
+		for(FrankAnnotation a: annotationsForLabels) {
+			try {
+				handleSpecificLabel(a, labelValues);
+			} catch(FrankDocException e) {
+				log.error("Could not parse label [{}] of [{}]", a.getName(), toString());
+			}
+		}
+	}
+
+	void handleSpecificLabel(FrankAnnotation labelAddingAnnotation, LabelValues labelValues) throws FrankDocException {
+		String label = labelAddingAnnotation.getAnnotation(LABEL).getValueOf(LABEL_NAME).toString();
```

Comment:
Eigenlijk is label toch ook een Feature?

Reply:
We kunnen er een feature van maken. Als we dat doen, dan zou je `@Category` bijvoorbeeld kunnen veranderen in het volgende:

```
package nl.nn.adapterframework.doc;

import static java.lang.annotation.ElementType.TYPE;

import java.lang.annotation.Documented;
import java.lang.annotation.Target;

/**
   * @ff.label Category
   *
   */
@Target(TYPE)
@Documented
public @interface Category {
	@LabelValue public String value();
}
```

Dit is niet heel moeilijk. Maar hebben we er wat aan?